tromega/lib/views/plans/filter_exercises.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:tromega/views/plans/add_exercise.dart';
+import 'package:tromega/views/plans/edit_training_view.dart';
+import 'package:tromega/views/in_training/track_training_view.dart';
+
+import '../../widgets/plan/exercise_container_plan.dart';
+import 'package:flutter/material.dart';
+import '../../widgets/bottom_menu.dart';
+import '../../widgets/shared/app_bar.dart';
+import 'package:tromega/data/trainingDay.dart';
+import 'package:tromega/data/filter.dart';
+import 'package:tromega/data/trainingPlan.dart';
+import 'package:multi_select_flutter/multi_select_flutter.dart';
+
+class FilterView extends StatefulWidget {
+  //Visualisierung von einem Tag eines Trainingsplans
+  FilterView({Key? key, required this.filter}) : super(key: key);
+  Filter filter;
+
+  @override
+  State<FilterView> createState() => _FilterViewState();
+}
+
+class _FilterViewState extends State<FilterView> {
+  List<String> _selectedMuscle = [];
+  List<String> _selectedEquipment = [];
+
+  static final List<String> _muscleSelection = [

```

Comment:
Weiß nicht wie sinnvoll es ist, aber vielleicht holt man sich die möglichen Werte hier dynamisch aus dem Back-end? Falls wir da nochmal weitere Übungen anpassen und etwas anderes auftaucht, müssten wir das nicht extra anpassen

Reply:
Hab ich mir auch gedacht, aber dachte ist unsicher, wenn wir das an Backend geben, dass die route am ende steht

Reply:
Ist natürlich auch richtig, dann lassen wir es erstmal so und gehen vielleicht nochmal in den Kontakt mit dem Backend.
Schreib dafür vielleicht einfach noch eine Feature-Request und wenn die das nicht schaffen, bleibt es einfach hard-coded

Habs nur nochmal unresolved, damit du es siehst :)